voyager-publish/test/test.js

```diff
@@ -81,12 +81,17 @@ describe('unit', () => {
       logger
     })
 
-    assert.deepStrictEqual(clientQueryParams, [
-      [1],
-      undefined,
-      [[]],
-      undefined
-    ])
+    assert.strictEqual(clientQueryParams.length, 6)
+    assert.strictEqual(clientQueryParams[0], undefined)
+    assert(Array.isArray(clientQueryParams[1]))
+    assert.strictEqual(clientQueryParams[1][0], 1)
+    assert(clientQueryParams[1][1] instanceof Date)
+    assert.strictEqual(clientQueryParams[2], undefined)
+    assert.strictEqual(clientQueryParams[3], undefined)
+    assert(Array.isArray(clientQueryParams[4]))
+    assert(clientQueryParams[4][0] instanceof Date)
+    assert.strictEqual(clientQueryParams[5], undefined)
```

Comment:
I have no idea what this block is asserting. If one of these assertions fails, I will have no idea what's wrong.

Could you please try to come up with a different way how to test this part, one that will be easier to understand when reading the code and to troubleshoot on a test failure.

Reply:
Before that, this was a deep strict equal. Since we have date objects in here now (which we don't control), I'm testing values individually.

Do you have a suggestion how to change this test? I'm happy removing it too.

Reply:
ping @bajtos

Reply:
In https://github.com/filecoin-station/voyager-api/pull/22, I deleted `test/test.js` and replaced it with two new files - `unit.test.js` and `integration.test.js`.

Can you please delete `test/test.js` in your pull request too?